presto-main/src/main/java/com/facebook/presto/sql/Serialization.java

```diff
@@ -82,4 +87,38 @@ public FunctionCall deserialize(JsonParser jsonParser, DeserializationContext de
             return (FunctionCall) rewriteIdentifiersToSymbolReferences(sqlParser.createExpression(jsonParser.getText()));
         }
     }
+
+    public static class VariableReferenceExpressionSerializer
```

Comment:
Curious: Is the efficiency the reason to have specialized JsonSerializer ? Cause otherwise it can also serialize to JSON like `{"name": "XXX", "type": ...}` right? (although quite cumbersome)

Reply:
in JSON, Map keys can only be a string. Since `VariableReferenceExpression` is used as map keys, it needs to be serialized to a string.